glean-core/metrics.yaml

```diff
@@ -238,9 +238,22 @@ glean.error:
     data_reviews:
       - https://bugzilla.mozilla.org/show_bug.cgi?id=1499761#c5
     notification_emails:
-      - telemetry-client-dev@mozilla.com
+      - glean-team@mozilla.com
     expires: never
     send_in_pings:
       - all_pings
     no_lint:
       - COMMON_PREFIX
+
+  preinit_tasks_timeout:
+    type: boolean
+    description:
+      Set to true if the tasks that are queued prior to Glean initialization
+      time out.
+    bugs:
+      - https://bugzilla.mozilla.org/1575848
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1575848#c5
```

Comment:
We kinda extend the population for this metric. This means it requires a new data review.

Reply:
Good call!

Reply:
[The original review](https://bug1575848.bmoattachments.org/attachment.cgi?id=9104882) already states the population as:
```
7) What populations will you measure?
All products using Glean.
```

Having checked with @chutten via IRC, moving the metric only extends the mechanical population and not the approved population so we should be covered by the original review.